WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -2,634 +2,677 @@
+                            android:id="@+id/quick_action_pages_button"
+                            style="@style/MySiteQuickActionButton"
+                            android:backgroundTint="@android:color/white"
+                            android:src="@drawable/ic_pages_white_24dp"/>
 
-            <!--Quick Start-->
-            <androidx.cardview.widget.CardView
-                android:id="@+id/quick_start"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/margin_large"
-                android:layout_marginEnd="@dimen/content_margin"
-                android:layout_marginStart="@dimen/content_margin"
-                android:layout_width="match_parent"
-                app:cardBackgroundColor="@android:color/white"
-                app:cardCornerRadius="@dimen/default_cardview_radius"
-                app:cardElevation="@dimen/card_elevation">
+                        <org.wordpress.android.widgets.WPTextView
+                            style="@style/MySiteQuickActionButtonLabel"
+                            android:text="@string/pages"/>
+                    </LinearLayout>
+
+                    <LinearLayout
+                        style="@style/MySiteQuickActionButtonContainer">
+
+                        <com.google.android.material.floatingactionbutton.FloatingActionButton
+                            android:id="@+id/quick_action_posts_button"
+                            style="@style/MySiteQuickActionButton"
+                            android:src="@drawable/ic_posts_white_24dp"/>
+
+                        <org.wordpress.android.widgets.WPTextView
+                            style="@style/MySiteQuickActionButtonLabel"
+                            android:text="@string/posts"/>
+                    </LinearLayout>
+
+                    <LinearLayout
+                        style="@style/MySiteQuickActionButtonContainer">
+
+                        <com.google.android.material.floatingactionbutton.FloatingActionButton
+                            android:id="@+id/quick_action_media_button"
+                            style="@style/MySiteQuickActionButton"
+                            android:src="@drawable/ic_media_white_24dp"/>
+
+                        <org.wordpress.android.widgets.WPTextView
+                            style="@style/MySiteQuickActionButtonLabel"
+                            android:text="@string/media"/>
+                    </LinearLayout>
+
+                </LinearLayout>
 
+                <!--Quick Start-->
                 <RelativeLayout
+                    android:id="@+id/quick_start"
+                    android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_width="match_parent" >
+                    android:layout_marginBottom="@dimen/margin_large"
+                    android:background="@color/white"
```

Comment:
```suggestion
                    android:background="@android:color/white"
```

Can we change this to `@android:color/white`?  There is no resource `white` in `colors.xml` so using `@color/white` would be pulling from another source or library.